src/index.tsx

```diff
@@ -1,10 +1,19 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import App from './app';
+import { Auth0Provider } from '@auth0/auth0-react';
 
 
 ReactDOM.createRoot(document.getElementById('root')!).render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
+
+  <Auth0Provider
```

Comment:
don't use providers and HOCs - we can use models and hooks directly in our components

Reply:
this is not ui code